### PR TITLE
InstantOn standalone GA post edits per review

### PR DESCRIPTION
**Pull request by dmuelle**
#3129 

post on draft: https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2023/06/27/rapid-startup-instanton.html

### CODE REVIEW

**Comment by vijaysun-omr** [comment]
"These are specific phases during Open Liberty startup that you can choose from and configure the Dockerfile needs to specify the phase (as we show later)." 

may be better worded as 

"These are specific phases during Open Liberty startup that you can choose from and you need to configure the Dockerfile to specify the phase (as we show later)."

**Comment by vijaysun-omr** [comment]
"The checkpoint.sh script allows you to specify either afterAppStart or beforeAppStart to indicate which phase of the startup performs the process checkpoint." 

may need to use different font for `checkpoint.sh`, `afterAppStart` and `beforeAppStart` as is done at other places in this doc.

**Comment by vijaysun-omr** [comment]
If you look at the performance graphs in the .docx file at [this Box location](https://ibm.ent.box.com/file/1217422291725?s=71qwjvxci6divqv6gtrynhn0ajax62lg), the graphs have a consistent scheme related to the colors used and the placement of numbers above the bars in the graph. This is not so in the preview that this PR shows. Can the formatting for the graphs be fixed so that it looks more like what is in the .docx file please ?

**Comment by vijaysun-omr** [comment]
"For more details about Open Liberty InstantOn, see the [Fast startup for containerized applications with Open Liberty InstantOn](https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2023/06/27/docs/latest/instanton.html) documentation, which links to more elaborate discussion on known limitations and information on the Semeru JDK support for this feature." in the summary section is a link that I assume will magically work when the article goes live (it led to a page not found error for me when I clicked on it).

**Comment by lauracowen** [comment]
Minor comment. I just took a look on the preview site and the start seems like a wall of text. Then the end has two consecutive images. Would it work to put the startup speed graph somewhere in the first section? Maybe following "InstantOn requires very little modification of your Java application to make this improvement happen." to break up the text. There's been mention by then of the 90% improvement so it's not completely out of context, what do you think? And it would make the article look more appealing at a glance (before reading anything).

Alternatively, any other image in that first section would be good.

**Comment by lauracowen** [comment]
> Alternatively, any other image in that first section would be good.

@dmuelle Vijay has created an image which I've sent to you on slack. Thank you for all your help on this!

File: posts/2023-06-27-rapid-startup-instanton.adoc [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+Then, build the application, which is in the `finish/` directory, and deploy it to Open Liberty:
+
+[source,console]
+----
+cd finish
+mvn liberty:run
+----
+
+When you see the following message, your application server is ready:
```
Comment (yeekangc) [inline posts/2023-06-27-rapid-startup-instanton.adoc]:
```suggestion
When you see the following message, your Open Liberty instance is ready:
```

File: posts/2023-06-27-rapid-startup-instanton.adoc [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+[source,console]
+----
+The defaultServer server is ready to run a smarter planet.
+----
+
+Check out the service at the http://localhost:9080/dev/system/properties URL.
+Stop the Open Liberty server by pressing **CTRL+C** in the command-line session where you ran the server.
```
Comment (yeekangc) [inline posts/2023-06-27-rapid-startup-instanton.adoc]:
```suggestion
Stop the Open Liberty server by pressing **CTRL+C** in the command-line session where you ran the server.
```
```suggestion
Stop the running Open Liberty instance by pressing **CTRL+C** in the command-line session where you started Open Liberty.
```

File: posts/2023-06-27-rapid-startup-instanton.adoc [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+[source,console]
+----
+podman run --name getting-started --rm -p 9080:9080 getting-started
+----
+
+Note the amount of time Open Liberty takes to report it is started and check out the service running in the container at the http://localhost:9080/dev/system/properties URL. After you finish checking out the application, stop the running container by pressing **CTRL+C** in the command-line session where you ran the `podman run` command.
+
+=== Containerizing the Open Liberty application with InstantOn
+
+The Open Liberty container image contains the prerequisites for building an application container image with a checkpointed server process. Applications can use the Open Liberty image as a base to build their own application container images and from that, create their own application container image with a checkpointed process.
```
Comment (yeekangc) [inline posts/2023-06-27-rapid-startup-instanton.adoc]:
```suggestion
The Open Liberty container image contains the prerequisites for building an application container image with a checkpointed runtime process. Applications can use the Open Liberty image as a base to build their own application container images and from that, create their own application container image with a checkpointed process. 
```

**Comment by dmuelle** [comment]
> "For more details about Open Liberty InstantOn, see the [Fast startup for containerized applications with Open Liberty InstantOn](https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2023/06/27/docs/latest/instanton.html) documentation, which links to more elaborate discussion on known limitations and information on the Semeru JDK support for this feature." in the summary section is a link that I assume will magically work when the article goes live (it led to a page not found error for me when I clicked on it).

@vijaysun-omr - yes, this preview is on the blogs draft site, which does not include docs content, so the link cannot resolve. Also, this particular doc will not be published until just before the blog post is published. I will verify the link before publishing.

**Comment by dmuelle** [comment]
Hi, I believe I've addressed all comments here- committing the updates to draft for review and will open another staging PR

**Pull request merged**